transport/http/handler_test.go

```diff
@@ -52,7 +52,7 @@ func TestHandlerSucces(t *testing.T) {
 	headers.Set(BaggageHeaderPrefix+"BAR", "baz")
 
 	rpcHandler := transporttest.NewMockHandler(mockCtrl)
-	httpHandler := handler{rpcHandler}
+	httpHandler := handler{rpcHandler, transport.NoDeps}
```

Comment:
Might be easier to just do `handler{Handler: rpcHandler}` since zero-value of `Deps` is now valid.